local/owr_video_renderer.c

```diff
@@ -311,10 +311,13 @@ static GstElement *owr_video_renderer_get_element(OwrMediaRenderer *renderer, gu
     renderer_disabled(renderer, NULL, balance);
 
     flip = gst_element_factory_make("glvideoflip", "video-renderer-flip");
-    g_assert(flip);
-    g_signal_connect_object(renderer, "notify::rotation", G_CALLBACK(update_flip_method), flip, 0);
-    g_signal_connect_object(renderer, "notify::mirror", G_CALLBACK(update_flip_method), flip, 0);
-    update_flip_method(renderer, NULL, flip);
+    if (!flip) {
+        g_warning("The glvideoflip element isn't available");
```

Comment:
I think this comment could probably be a bit more human-readable as to the consequences (that mirroring and rotation functionality will not work.)